crates/aptos-workspace-hack/Cargo.toml

```diff
@@ -41,6 +41,8 @@ itertools = { version = "0.10.3", features = ["use_alloc", "use_std"] }
 libc = { version = "0.2.123", features = ["std"] }
 log = { version = "0.4.16", default-features = false, features = ["serde", "std"] }
 memchr = { version = "2.4.1", features = ["std", "use_std"] }
+move-unit-test = { git = "https://github.com/move-language/move", rev = "476305d239ce6afafce15a297c8c3839dd8465b6", default-features = false, features = ["table-extension"] }
+move-vm-test-utils = { git = "https://github.com/move-language/move", rev = "476305d239ce6afafce15a297c8c3839dd8465b6", features = ["move-table-extension", "table-extension"] }
```

Comment:
why there's two features?

Reply:
There isn't really a "move-table-extension" feature.. I think it's a bug in x..